chia/_tests/blockchain/blockchain_test_utils.py

```diff
@@ -73,19 +72,22 @@ async def _validate_and_add_block(
     new_slot = len(block.finished_sub_slots) > 0
     ssi, diff = get_next_sub_slot_iters_and_difficulty(blockchain.constants, new_slot, prev_b, blockchain)
     await check_block_store_invariant(blockchain)
+
     if skip_prevalidation:
-        results = PreValidationResult(None, uint64(1), None, False, uint32(0))
+        if block.transactions_generator is None:
+            conds = None
+        else:
+            # fake the signature validation. Just say True here.
+            conds = SpendBundleConditions([], 0, 0, 0, None, None, [], 0, 0, 0, True)
+        results = PreValidationResult(None, uint64(1), conds, True, uint32(0))
```

Comment:
two things to note here:
1. The faking of having results is improved by creating a dummy `SpendBundleConditions` object
2. Both `SpendBundleConditions` and `PreValidationResult` say the signatures were validated.

(2) is because they are now expected to have been validated by the time you call `validate_block_body()`, and this is `assert`ed. Previously you could ask `validate_block_body()` not to validate, but that option has been removed.